pandas/tests/io/json/test_pandas.py

```diff
@@ -1032,6 +1038,102 @@ def test_to_jsonl(self):
         assert result == expected
         assert_frame_equal(pd.read_json(result, lines=True), df)
```

Comment:
the lines=Trye test could go in a separate test class (you can move the existing ones)

Reply:
Will do. While we're at it, what do you think of the name of the existing test class (`TestPandasContainer`)? I want them to be consistent, but I don't really understand the name of the existing one.

Reply:
sure i think that was copied from the actual json tests (so should be more reflective);

also ok with a separate file and no class (iow bare functions); this was just the existing structure ; we can/should use fixtures